Dockerfile.assisted-service-build

```diff
@@ -6,7 +6,7 @@ ENV PATH=$PATH:$GOROOT/bin:$GOPATH/bin
 
 USER 0
 
-RUN yum install -y epel-release && yum install -y gcc genisoimage jq
+RUN yum install -y epel-release && yum install -y gcc genisoimage jq git
 
 COPY --from=registry.ci.openshift.org/openshift/release:golang-1.16 /usr/bin/git /usr/bin/gotestsum /usr/bin/make /usr/bin/
```

Comment:
If we are installing git then we can skip copying the git from the golang layer.

Nit:
`COPY --from=registry.ci.openshift.org/openshift/release:golang-1.16 usr/bin/gotestsum /usr/bin/make /usr/bin/`

Reply:
good idea
thanks!